# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/oh-my-posh/gruvbox.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Make it more obvious that we are root when using the gruvbox theme.
Also enable the subversion segment.

  - NoTag <notag@maneto.fr>

diff --git a/themes/gruvbox.omp.json b/themes/gruvbox.omp.json
index 9013c925..6981563e 100644
--- a/themes/gruvbox.omp.json
+++ b/themes/gruvbox.omp.json
@@ -13,6 +13,9 @@
         },
         {
           "background": "#458588",
+          "background_templates": [
+            "{{ if .Root }}#FF6568{{ end }}"
+          ],
           "foreground": "#282828",
           "powerline_symbol": "\ue0b0",
           "properties": {
@@ -45,6 +48,16 @@
           "trailing_diamond": "\ue0b4",
           "type": "git"
         },
+	{
+          "background": "#98971A",
+          "foreground": "#282828",
+          "powerline_symbol": "\ue0b0",
+          "properties": {
+            "style": "full"
+          },
+          "style": "powerline",
+          "type": "svn"
+	},
         {
           "background": "#8ED1F7",
           "foreground": "#111111",
@@ -116,14 +129,6 @@
           "style": "powerline",
           "template": " \ue7ad {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} ",
           "type": "aws"
-        },
-        {
-          "background": "#ffff66",
-          "foreground": "#111111",
-          "powerline_symbol": "\ue0b0",
-          "style": "powerline",
-          "template": " \uf0ad ",
-          "type": "root"
         }
       ],
       "type": "prompt"
